server/studyspaces.py

```diff
@@ -57,57 +27,8 @@ def parse_times(date):
         return jsonify({'studyspaces': m})
 
 
-def extract_times(id, date, name):
-    """
-    Scrapes the avaiable rooms with the given ID and date.
-    """
-    url = "http://libcal.library.upenn.edu/rooms_acc.php?gid=%s&d=%s&cap=0" % (int(id), date)
-    soup = BeautifulSoup(requests.get(url).text, 'lxml')
-
-    timeSlots = soup.find_all('form')
-    unparsedRooms = timeSlots[1].contents[2:-2]
-
-    roomTimes = []
-
-    for i in unparsedRooms:
-        room = BeautifulSoup(str(i), 'lxml')
-        try:
-            # extract room names
-            roomName = room.fieldset.legend.h2.contents[0]
-        except AttributeError:
-            # in case the contents aren't a list
-            continue
-        newRoom = str(roomName)[:-1]
-        times = []
-
-        filt = room.fieldset.find_all('label')
-
-        for t in filt:
-            # getting the individual times for each room
-            dictItem = {}
-            dictItem['room_name'] = newRoom
-            time = str(t).split("\t\t\t\t\t")[1][1:-1]
-            times.append(time)
-            startAndEnd = time.split(" - ")
-            dictItem['start_time'] = startAndEnd[0].upper()
-            dictItem['end_time'] = startAndEnd[1].upper()
-            roomTimes.append(dictItem)
-            dictItem['date'] = date_parse(date)
-            dictItem['building'] = name
-    return roomTimes
-
-
-def date_parse(d):  
-    """
-    Parses the date to dashed format.
-    """
-    l = d.split("-")
-    final = [l[1], l[2], l[0]]
-    return '-'.join(final)
-
-
-@app.route('/studyspaceid', methods=['GET'])
-def display():
+@app.route('/studyspaces/ids', methods=['GET'])
```

Comment:
REST URL proposal: `/studyspaces/` since this lists all the available studyspaces